src/module/freebsd/freebsd.c

```diff
@@ -0,0 +1,91 @@
+#include "freebsd.h"
+#include "../targeting.h"
+#include <sys/proc.h>
+#include <sys/lock.h>
+#include <sys/mutex.h>
+#include <sys/param.h>
+#include <sys/vnode.h>
+#include <sys/file.h>
+#include <sys/filedesc.h>
+
+bool krf_personality(unsigned int target, krf_ctx_t *context) {
```

Comment:
FreeBSD provides `__always_inline` like Linux, so we can mark these like their Linux counterparts.